README.md

```diff
@@ -2,25 +2,22 @@
 
 A [Concourse](http://concourse.ci/) [resource](http://concourse.ci/resources.html) to interact with the [GitHub Status](https://developer.github.com/v3/repos/statuses/) type.
 
-
 ## Configuration
 
  * **`repository`** - the owner and repository name, slash delimited (e.g. `dpb587/github-status-resource`)
  * **`access_token`** - GitHub API access token from a user with write access to the repository (minimum token scope of `repo:status`)
  * `branch` - the branch currently being monitored (default: `master`)
+ * `base_context` - prefix for the context label (default: `concourse-ci`)
```

Comment:
Can you describe the use case for this a bit more?

Assuming it makes sense, I don't think it should be defaulted (since it'd be a large, breaking change). So I think this should be adjusted to actually be the full prefix (where the user sets it to `concourse-ci/` if they want this behavior) and then remove the slashes deeper in code.

Reply:
There are a few reasons for this change:

- it will make this resource consistent with the `git` and `github-pr-resources`, providing a more unified user experience across Concourse
- we have a staging cluster for testing new versions of Concourse, this allows us to have pipelines on this staging cluster use a prefix of `concourse-staging-ci`
- if you are testing changes to a pipeline before applying them to the primary pipeline, you can use this to differentiate between the two, e.g. `concourse-ci/dev`

Reply:
W.r.t. breaking change, we could just generate a new `2.2.0` tag for the current version, then merge and generate a `3.0.0` tag. By not having the `concourse-ci` be the default, then it would deviate from the `github-pr-resource`. 🤔